.github/workflows/generate.yaml

```diff
@@ -4,6 +4,10 @@ on:
   workflow_dispatch:
   schedule:
     - cron: '5 4 * * SUN'
+  push:
+    branches: [ main ]
+    paths:
+      - ./.github/workflows/assets/swagger.yaml
```

Comment:
For the time being I thought it would be nice to have this action automatically trigger if we change the `swagger.yaml` file on `main`.